services/ad-tech/src/public/js/ssp/ssp-tag.js

```diff
@@ -17,54 +20,205 @@
  *   setup. This is included on certain tagged publisher pages.
  *
  * What does this script do:
- *   This script injects an iframe in its own origin to invoke runAdAuction()
- *   and renders the ad in a fenced frame.
+ *   This script orchestrates the loading of other seller modules to deliver
```

Comment:
How are ssp-a and ssp-b going to be differentiated? Or is the assumption that ssp-a and ssp-b are technically exactly the same, and we only have them to show multi-seller auction functionality?  Same question for dsp-a and dsp-b

Reply:
They are not differentiated currently, and they're mostly needed for the multi-seller setup. Since it's the same docker image talking to another instance of itself, I'm mentally modeling this like a kaleidoscope where multiple layers of use-cases in each instance reflect on each other to make it seem like a lot is going on.